### ISSUE TITLE
[Portal] Gravitee Login page Error: Server unreachable

### ISSUE THREAD

**Issue by chico35400** [gravitee-io/issues#5124]
Hello,
we have a « Server unavailable or connection lost » error on Gravitee Portal.
We can reach our gravitee services on Ports 8082/8083 from outsite (via curl)
our constans.json seams to be ok :
_{
  "baseURL": "https://api-mgmt-test-3-6.domaine.com/management/"
}_
But we have CORS and 404 errors in clients.
![image](https://user-images.githubusercontent.com/77109288/108874997-09693e80-75fd-11eb-8eb7-43072af53377.png)

Do we have to declare other vhosts  8082/8083  in apache too like vhost :80?

_<VirtualHost *:80>
        ServerAdmin admin_unix@domaine.net
        ServerName api-ui-test-3-6.domaine.com
        ErrorLog /var/log/apache2/error-api-ui-test-3-6.domaine.com.log
        CustomLog /var/log/apache2/access-api-ui-test-3-6.domaine.com.log common
        DocumentRoot /usr/local/graviteeio-full-3.6.0/graviteeio-portal-ui-3.6.0
        <Directory /usr/local/graviteeio-full-3.6.0/graviteeio-portal-ui-3.6.0>
                Options FollowSymLinks MultiViews
                AllowOverride All
                Require all granted
                RewriteEngine on
        </Directory>
</VirtualHost>_

thanks for help
regards

**Comment by aelamrani** [comment]
Hi @chico35400 

Yes you have to configure CORS on your management API component.

You can refer to the following documentation for that: https://docs.gravitee.io/apim/3.x/apim_installguide_rest_apis_configuration.html#cors_configuration

**Comment by chico35400** [comment]
Thanks for the answer.

We have configured CORS, but we still have the « Server unavailable or connection lost » error on Portal.
We tried too with "allow-origin: *" for tests purposes: same result
We tried to build a default install (default config , with all modules on the same server, gw, api, managment, portal, elastic, mongo and apache): same result too.
we see no errors on services lauch
Gravitee services on Ports 8082/8083 from outsite are still reachables via curl
but the portal on the same server fail to connect to them.

any other clue?
